Sources/Model/ObjC/Common/Core Data/TracksContextManager.m

```diff
@@ -81,7 +81,7 @@ - (NSURL *)applicationSupportURLForAppWithBundleIdentifier:(NSString *)bundleIde
     // It seems safe not to handle this error because Application Support should always be
     // available and one should always be able to create a folder in it
     if (error != nil) {
-        TracksLogError(@"Failed to create folder for %@ in Application Support: %@, %@", bundleIdentifier, error, [error userInfo]);
+        TracksLogPanic(@"[Tracks] ApplicationSupport Creation for %@ Error: %@, %@", bundleIdentifier, error, [error userInfo]);
```

Comment:
Should we remove the abort below? It feels like The app doesn't need to crash just because Tracks could not be initialized?

Reply:
I would keep the abort() call, since ... Tracks will not be functional (and thus, in an inconsistent state).

If we intend to remove the abort invocation, we should, then, build (Something similar) to a failable initializer. Otherwise the apps using this will not be aware the internal state is broken.

Reply:
Okay, I don't have strong feelings on it I guess? @bjhomer , do you have thoughts?

Reply:
I agree that we shouldn't kill the entire app. A throwing initializer would probably be appropriate.